mlflow/pytorch/__init__.py

```diff
@@ -363,11 +368,17 @@ def _load_model(path, **kwargs):
     :param kwargs: Additional kwargs to pass to the PyTorch ``torch.load`` function.
     """
     import torch
+    pytorch_version = torch.__version__
 
     if os.path.isdir(path):
         # `path` is a directory containing a serialized PyTorch model and a text file containing
         # information about the pickle module that should be used by PyTorch to load it
         model_path = os.path.join(path, "model.pth")
+
+        local_path = os.path.dirname(path)
+        pytorch_conf = _get_flavor_configuration(model_path=local_path, flavor_name=FLAVOR_NAME)
+        pytorch_version = pytorch_conf["pytorch_version"]
```

Comment:
Why do we need to use the pytorch version used to saves the model?

Reply:
https://github.com/chauhang/mlflow/blob/1eec412236ae8f06c2dabd2dde28175ed6bb06a1/tests/pytorch/test_pytorch_model_export.py#L438 - This test specifically alters the pytorch version and without this condition, pytorch version would be set to 1.6 and  loading the model using `torch.load`  results in exception.

Reply:
I think this issue can be fixed like this:
: https://github.com/harupy/mlflow/commit/cba164e1989e7857170a26752538a31bd94fd7c5

Reply:
Thanks . Let me check.